tests/cmdline/commands/test_archive_import.py

```diff
@@ -225,7 +225,7 @@ def test_raise_malformed_url(self):
         self.assertIsNotNone(result.exception, result.output)
         self.assertNotEqual(result.exit_code, 0, result.output)
 
-        error_message = 'Is it a valid path or URL?'
+        error_message = 'could not be reached within'
```

Comment:
This was one of the changes I stumbled over. It appears like an unrelated change.